webapp/components/wpt-results.html

```diff
@@ -537,6 +587,12 @@
 
         console.log('JSON version:', JSON.stringify(totals));
       }
+
+      showHistoryClicked() {
+        return () => {
+          this.showHistory = true;
+        };
```

Comment:
Not a dumb question; the lambda doesn't bind `this`, while calling a function does (bind `this` to the element that was clicked). To be able to refer back to the custom element, rather than the button, we return an arrow function.

Reply:
Ahh, I see, same as `bindNavigation` I saw somewhere else. Thx!